### ISSUE TITLE
Wont extract/work (?)

### ISSUE THREAD

**Issue by TrumpetTrash** [pokerobybeto/Ultimate-Stream-Tool#1]
Been using this for a while now and all of a sudden, it just stopped working.
When I tried to re-download it, it says that a certain file won't work and im not able to extract it to use it. 
Please help.

**Comment by Sukid** [comment]
Did you update OBS to 28? all of the smash overlays broke with that one.

**Comment by TrumpetTrash** [comment]
Yes I think that might be the problem then. I'll see if I can go back to a
later version and let you know if that works.
